deployments/fpga_plugin/build-initcontainer-image.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh -xe
+
+# build CRI hook
+HOOK=fpga_crihook
+make -C ../../ $HOOK
+cp ../../cmd/$HOOK/$HOOK ./intel-fpga-crihook
+
+# build initcontainer image
+WORKSPACE=`realpath .`
+IMG="intel-fpga-initcontainer"
+TAG=$(git rev-parse HEAD)
+
+BUILD_ARGS='--rm'
+for var in ftp_proxy http_proxy https_proxy; do
+    BUILD_ARGS="${BUILD_ARGS} --build-arg $var=`echo ${!var}`"
```

Comment:
Don't do that. First of all, variables can be upper or lower case. Second, no need to expand value. Just `--build-arg $var` is enough. However, this method anyway should not be used anymore.
Proxy in current versions of docker should be configured via `~/.docker/config.json`.